docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
+----------------
+
+.. _collection-list:
+
+.. note:: This API requires :doc:`authentication <auth>`.
+
+This endpoint allows you to list all collections authored by this user.
+Access is limited to your own collections, or users with `Users:Edit` permission.
+
+.. http:get:: /api/v3/accounts/account/(int:user_id|string:username)/collection/
+
+    :>json int count: The number of results for this query.
+    :>json string next: The URL of the next page of results.
+    :>json string previous: The URL of the previous page of results.
+    :>json array results: An array of :ref:`collections <collection-detail-object>`.
+
+
+-----------------
+Collection Detail
+-----------------
+
+.. _collection-detail:
+
+This endpoint allows you to fetch a single collection by its `slug`.
+Access to non-`listed` collections are limited to the author, or users with `Users:Edit` permission
+
+.. http:get:: /api/v3/accounts/account/(int:user_id|string:username)/collection/(string:collection_slug)/
+
+    .. _collection-detail-object:
+
+    :>json int id: The id for the collection.
+    :>json int addon_count: The number of add-ons in this collection.
+    :>json int author.id: The id of the author (creator) of the collection.
+    :>json string author.name: The name of the author.
+    :>json string author.url: The link to the profile page for of the author.
+    :>json string description: The description the author added to the collection.
+    :>json string modified: The date the collection was last updated.
+    :>json string name: The of the collection.
+    :>json string url: The (absolute) collection detail URL.
+
+
+------------------
+Collection Add-ons
+------------------
+
+.. _collection-addon:
+
+This endpoint lists the add-ons in a collection, together with collector's notes.
+
+.. http:get:: /api/v3/accounts/account/(int:user_id|string:username)/collection/(string:collection_slug)/addons/
+
+    .. _collection-addon-object:
```

Comment:
I don't understand this response. Did you mean that this object is returned as each item in in the `results` key of the response (which is an array)? Also, I would have expected this response to include standard paging keys like `count`, `next`, and `previous`.